.github/workflows/documentation.yml

```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on: [release]
```

Comment:
push to master maybe? I think the doc should be up to date with GitHub version, @FabioLuporini @ggorman ?

Reply:
on:
   # Trigger the workflow on push or pull request,
   # but only for the master branch
   push:
     branches:
       - master